docs/help-center/using-famedly/advanced-features.mdx

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_label: Advanced features
+title: Advanced features
+hide_title: true
+displayed_sidebar: helpcenterSidebar
+sidebar_position: 8
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<div class="hero hero--primary">
+  <div class="container">
+    <h2 class="hero__title">Advanced features</h2>
+    <p class="hero__subtitle">We have some advanced features.</p>
+    <p>We want to test these features first before implementing complicated UI components.</p>
+  </div>
+</div>
+
+## General
+
+All advanced features are available via chat commands. You can enter the commands in the textfield of a chatroom and press send.
+
+### Change history visibility of a room
+
+The user needs to be admin of a room.
+
+- `/setHistoryVisibility <invited/joined/shared/worldReadable>` -> This changes the history visibility in this room.
+
+### Ignore users
+
+- `/ignore <@mxid>` -> This adds a user to the ignore list.
+- `/unignore <@mxid>` -> This removes the user from the ignore list.
+- `/ignoreList` -> This command posts the list of ignored users into the notes room..
+
+### Get reported events
+
+<b>The user needs to be OrgAdmin.</b>
```

Comment:
Maybe we should also mention that user needs to be on a server with this feature enabled.